ci/pipelines/Build.jenkinsfile

```diff
@@ -0,0 +1,93 @@
+def channel = "testing"
+def version = "0.0.0"
+def packageName = "DbPostgresAdapter"
+def testApplication = "DbPostgresAdapterTest"
+
+library identifier: "cpp-jenkins-pipelines@master", retriever: modernSCM(
+  [$class: "GitSCMSource",
+   remote: "https://github.com/systelab/cpp-jenkins-pipelines.git",
+   credentialsId: "GitHubCredentials"])	
+   
+pipeline
```

Comment:
**Not for now** but we need to give it a second though on how we want to handle these kind of libraries in terms CI/CD. I think we should probably have some kind of proxy repo in our private Bitbucket to hold all the pipelines and related files as to how to build these libraries, while having source code available to everyone.

Reply:
This will take some time, but it could be a good improvement